tests/compile-fail/chdir_invalid_path.rs

```diff
@@ -0,0 +1,11 @@
+// compile-flags: -Zmiri-disable-isolation
+
+extern {
+    pub fn chdir(dir: *const u8) -> i32;
+}
+
+fn main() {
+    let path = vec![0xc3u8, 0x28, 0];
+    // test that `chdir` errors with invalid utf-8 path
```

Comment:
But why should it error? Linux paths are not required to be valid UTF-8 and Rust can generally represent that (via `OsStr`).

Reply:
Removed on https://github.com/rust-lang/miri/pull/993